source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,37 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-03-24
+
+### Improvements
+
+#### User Interface (UI)
+ - Added support for automatic right-to-left (RTL) detection in browsers.
+ - Updated the font size for the **Add People** channel modal.
+ - Online status is now shown in the channel switcher.
+ - Improved the design and layout of email notifications.
```

Comment:
let's be specific about which types of email notifications were updated, not all types were. It should be in the PR

Reply:
Updated.